finality-aleph/src/lib.rs

```diff
@@ -95,15 +95,17 @@ fn max_message_size(protocol: Protocol) -> u64 {
 pub fn peers_set_config(
     naming: ProtocolNaming,
     protocol: Protocol,
-) -> sc_network::config::NonDefaultSetConfig {
-    let mut config = sc_network::config::NonDefaultSetConfig::new(
+) -> (
+    sc_network::config::NonDefaultSetConfig,
+    Box<dyn sc_network::config::NotificationService>,
+) {
+    sc_network::config::NonDefaultSetConfig::new(
         naming.protocol_name(&protocol),
+        naming.fallback_protocol_names(&protocol),
         max_message_size(protocol),
-    );
-
-    config.set_config = sc_network::config::SetConfig::default();
-    config.add_fallback_names(naming.fallback_protocol_names(&protocol));
-    config
+        None,
```

Comment:
Maybe comment that this is the handshake, since it's not quite obvious.

Reply:
done